.github/workflows/pr_views_to_core.yml

```diff
@@ -0,0 +1,38 @@
+name: PR Views to PrefectHQ/prefect
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "views/aggregate-worker-metadata.json"
+  workflow_dispatch:
```

Comment:
this looks off to me - i thought `workflow_dispatch` was an event, e.g., `on: workflow_dispatch`

Reply:
I think that is what I have here, it should run `on:` `push` to `main` or `on:` `workflow_dispatch`

[e.g. this one from core](https://github.com/PrefectHQ/prefect/blob/c6ef1e633960f0b9167c68737eddc06907af64f5/.github/workflows/docker-images.yaml#L3-L11)

Reply:
Gotcha gotcha, I guess the trailing semicolon threw me off - thank you!